feat: added configurable keybindings to alter config values at run time

ramirezmike opened the pull request:
I added some config bindings to flip/increment/decrement a few config settings in order to setup something like "Alt+I" toggles display of the appbar. I tend to setup i3 on my linux box this way and kept wanting to hide the bar when not in use but also I found in windows sometimes I do want to see the task bar, like for altering stuff in the tray icons window, so this also let me show/hide the task bar without editing my config file. Figured it can be used for other things like adjusting sizes of things too. Let me know what you think.

TimUntersberger commented on src/config.rs:
```diff
@@ -104,6 +104,39 @@ impl Config {
+            "app_bar_font_size" => self.app_bar_font_size += value,
+            "margin" => self.margin += value,
+            "padding" => self.padding += value,
+            _ => debug!("Attempt to alter unknown field: {} by {}", field, value)
```
I feel like this should be more an `error!` than a `debug!`.
```suggestion
            _ => error!("Attempt to alter unknown field: {} by {}", field, value)
```

TimUntersberger commented on src/config.rs:
```diff
@@ -104,6 +104,39 @@ impl Config {
+            "remove_title_bar" => self.remove_title_bar = !self.remove_title_bar,
+            "remove_task_bar" => self.remove_task_bar = !self.remove_task_bar,
+            "display_app_bar" => self.display_app_bar = !self.display_app_bar,
+            _ => debug!("Attempt to toggle unknown field: {}", field)
```
Same thing here.
```suggestion
            _ => error!("Attempt to toggle unknown field: {}", field)
```

ramirezmike commented:
Agreed, updated. Also, I removed work_mode and multi_monitor as work_mode has a dedicated keybinding and I think multi_monitor might require additional work in the hot reload to work correctly.

TimUntersberger commented:
> I think multi_monitor might require additional work

Nice catch!

**LGTM**
